IPs/RSKIP383.md

```diff
@@ -0,0 +1,64 @@
+---
+rskip: 383
+title: POWpeg migration age normalization
+created: 10-MAY-23
+author: JD
+purpose: Fair
+layer: Core
+complexity: 1
+status: Draft
+description: 
+---
+
+|RSKIP          |383           |
+| :------------ |:-------------|
+|**Title**      |POWpeg migration age normalization |
+|**Created**    |10-MAY-23 |
+|**Author**     |JD |
+|**Purpose**    |Fair |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+## Abstract
+
+Everytime the POWpeg composition changes a migration takes place. This migration consists of three stages:
+- Preparation: A new POWpeg has been voted but the new POWpeg has no control yet.
+- Activation: The new POWpeg is now active but the funds from the retiring POWpeg have not started to migrate yet.
+- Migration: The UTXOs from the retiring POWpeg are migrated to the active POWpeg
+
+Each of these stages have a certain duration set in Rootstock blocks to enable users and/or systems to act on them.
```

Comment:
```suggestion
Each of these stages has a certain duration set in Rootstock blocks to enable users and/or systems to act on them.
```

Reply:
As we are talking about more than one stage, _they have_ is the correct spell